src/tests/unittests/validation/CopyCommandsValidationTests.cpp

```diff
@@ -114,89 +114,110 @@ class CopyCommandTest_B2T : public CopyCommandTest {
 
 // Test a successfull B2T copy
 TEST_F(CopyCommandTest_B2T, Success) {
-    nxt::Buffer source = CreateFrozenBuffer(16 * 4, nxt::BufferUsageBit::TransferSrc);
-    nxt::Texture destination = CreateFrozen2DTexture(16, 16, 5, nxt::TextureFormat::R8G8B8A8Unorm,
+    nxt::Buffer source = CreateFrozenBuffer(256 * 4, nxt::BufferUsageBit::TransferSrc);
+    nxt::Texture destination = CreateFrozen2DTexture(256, 16, 5, nxt::TextureFormat::R8G8B8A8Unorm,
```

Comment:
The size of the texture doesn't need to be changed, only the buffer needs to have enough space for the extra padding. All of the texture related arguments below can stay to their original values and rowPitch can be left to 0 in most cases.